### DISCUSSION TITLE
No Internet =(

### DISCUSSION THREAD

**Discussion by chicopolding** [utmapp/UTM#2681] [UTM Help (macOS)]
Hello,

the following problem here. On a new M1 iMac 24" with macOS 11.4 I have no internet connection with UTM 2.1.2 (31) VM's. I have tested Ubuntu and ArchLinux (arm64) and no distribution came to the internet.
Is there anything else to consider here ?

Many thanks already for the help.

**Comment by saimurali619** [comment]
> Hello,
> 
> the following problem here. On a new M1 iMac 24" with macOS 11.4 I have no internet connection with UTM 2.1.2 (31) VM's. I have tested Ubuntu and ArchLinux (arm64) and no distribution came to the internet.
> Is there anything else to consider here ?
> 
> Many thanks already for the help.

Use DNS as 8.8.8.8 or 1.1.1.1 in Ubuntu/Linux VM and leave the ip settings automatically to system to be automatically assigned by VM.

**Comment by alxiong** [comment]
I encountered the same problem: MacBook Pro (13-inch, M1, 2020) M1 on archlinux-arm64-utm. 
Don't even have [`iwd`](https://wiki.archlinux.org/title/Iwd#iwctl) or `iwctl` installed to connect to wireless network.

**Reply by ktprograms** [reply]
Virtual machines don't have the wireless hardware exposed to them. The network is emulated as an ethernet card. 

As for Arch Linux, you can follow my guide at https://ktprograms.codeberg.page/blog/posts/2022-03-17_1750_utm-arch-arm/.
